03-task-manager/starter/models/Task.js

```diff
@@ -1,8 +1,15 @@
 const mongoose = require('mongoose');
 
 const TaskSchema = new mongoose.Schema({
-    name:String, 
-    completed:Boolean,
+    name:{
+        type:String, 
+        required:[true,'Must provide a name'], 
+        trim:true, 
+        maxlength:[20, 'Name cannot be more than 20 characters']
+    }, 
+    completed:{
+        type:Boolean, 
+        default:false},
```

Comment:
There's some inconsistency in the code formatting throughout; it's worth it to get something like prettier set up to go ahead and have something standard that triggers whenever you save your files.

One common solution is to use a dev. dependency, such as prettier:
[https://prettier.io/docs/en/install](https://prettier.io/docs/en/install)

I know that you can also set it up even more easily through an extension in VS Code if that's the editor you're using!
[https://www.freecodecamp.org/news/how-to-use-prettier-in-visual-studio-code/](https://www.freecodecamp.org/news/how-to-use-prettier-in-visual-studio-code/)